testing/trino-tests/src/test/java/io/trino/tests/TestMinWorkerRequirement.java

```diff
@@ -110,16 +111,11 @@ public void testInsufficientWorkerNodesAfterDrop()
             queryRunner.execute("SELECT COUNT(*) from lineitem");
             assertEquals(queryRunner.getCoordinator().refreshNodes().getActiveNodes().size(), 4);
 
-            try {
-                // Query should not be allowed to run if active workers drop down below the minimum required nodes
-                queryRunner.getServers().get(0).close();
-                assertEquals(queryRunner.getCoordinator().refreshNodes().getActiveNodes().size(), 3);
-                queryRunner.execute("SELECT COUNT(*) from lineitem");
-                fail("Expected exception due to insufficient active worker nodes");
-            }
-            catch (RuntimeException e) {
-                assertEquals(e.getMessage(), "Insufficient active worker nodes. Waited 1.00ns for at least 4 workers, but only 3 workers are active");
-            }
+            queryRunner.getServers().get(0).close();
+            assertEquals(queryRunner.getCoordinator().refreshNodes().getActiveNodes().size(), 3);
+            assertThatThrownBy(() -> queryRunner.execute("SELECT COUNT(*) from lineitem"))
+                    .isInstanceOf(RuntimeException.class)
+                    .hasMessage("Insufficient active worker nodes. Waited 1.00ns for at least 4 workers, but only 3 workers are active");
         }
     }
```

Comment:
shall we put lines 114,115 inside lambda as they have been in try {} before?

Reply:
@MariyaR I've refactored the code as suggested.